generators/kubernetes-helm/templates/app/requirements.yml.ejs

```diff
@@ -0,0 +1,22 @@
+dependencies:
+<%_ if (app.prodDatabaseType === 'mysql') { _%>
+  - name: mysql
+    version: ^0.10.1
```

Comment:
Why are the versions set up like that ? Especially the `^` sign ? I think we need to fix the versions otherwise it won't work half of the time.

Reply:
I see that there is a requirement.lock file which is created after deploy, so maybe it's OK but still it would be better to extract the versions to constants.

Reply:
Yep, as its more about getting the latest of minor and patch versions, there will not be any breaking changes to the services. Moreover intention of offloading the infra components is to run with the latest and stable releases from helm repo. Thought it would be better this way rather than limiting to a specific version